portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -89,23 +89,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     int numQueries = getLimit(request);
-    List<Entity> limitedResults = results.asList(FetchOptions.Builder.withLimit(10));
-    if(numQueries != -1){
-        limitedResults = results.asList(FetchOptions.Builder.withLimit(numQueries));
+    int defaultLimit = 10; 
+  
+    if(numQueries < 0 || numQueries > 10){
```

Comment:
so, really small suggestion here, but numQueries > 10 is still a pretty magic number. We might want to have another named variable like maxQueries, but if defaultLimit and maxQueries are always going to be the same number, we could probably just call them one thing and use it in all places.